.buildkite/pipeline.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - label: "Publish Simperium"
```

Comment:
For now I've only added a step for publishing, but in the long term, I think we'll want to add unit tests, lint and if possible connected tests as a dependency.

Reply:
AFAIK, simperium runs all tests (including unit tests) in their own suite that runs through connected tests.

Reply:
Does that include unit tests for this repo? That's quite interesting.

Reply:
Yes, I think the connected tests works as the test suite that loads the unit tests.